test/test_nn.py

```diff
@@ -11975,6 +11975,25 @@ def test_softmax_results(self, device, dtype):
                         self.assertEqual(grad_input, ref_grad_input)
                         self.assertEqual(input.grad, ref_input.grad)
 
+    @onlyCUDA
+    @dtypesIfCUDA(torch.float, torch.half)
+    @largeTensorTest("20GB")
```

Comment:
On my 3090, half takes ~18GB mem, and float takes ~19.8GB

Reply:
Will these tests run in your CI?

Reply:
Our CI has A100 and 3090, so yes!